pkg/cli/server.go

```diff
@@ -100,7 +100,7 @@ func (c *ServerCommand) RunUnstarted(ctx context.Context, args []string) (*plugi
 	ghAppCfg := githubapp.NewConfig(c.cfg.GitHubAppID, c.cfg.GitHubAppInstallationID, pk)
 	ghApp := githubapp.New(ghAppCfg)
 
-	p := plugin.NewGitHubPlugin(ctx, ghClient, ghApp)
+	p := plugin.NewGitHubPlugin(ctx, ghClient, ghApp, c.cfg)
```

Comment:
`DisplayName` and `Hit` are obtained in cfg from env var. Instead of passing using `displayName, hint string`, I think passing cfg makes more sense for future scalibility.